src/libmeasurement_kit/net/connect.hpp

```diff
@@ -31,13 +31,6 @@ void connect_first_of(Var<ConnectResult> result, int port,
                       Var<Logger> logger = Logger::global(), size_t index = 0,
                       Var<std::vector<Error>> errors = nullptr);
 
-typedef std::function<void(ResolveHostnameResult)> ResolveHostnameCb;
```

Comment:
This is the only part of code I removed when moving code. I removed it in favour of `Callback<ResolveHostnameResult> cb`